external-crates/move/crates/move-package/src/resolution/dependency_graph.rs

```diff
@@ -962,6 +1033,7 @@ impl DependencyGraph {
                 // the way that external graphs are constructed, edges between the (root) package of
                 // the outer graph and dependencies in the sub-graph are already present in the
                 // sub-graph
+                // TODO(rvantonder): this behavior needs to change to support "_" as a multi-dependency specifier
```

Comment:
Resolving external dependencies currently hardcodes that any dep declaration corresponds to a single valid dependency. This needs to be reworked to support adding multiple `dep_pkg_id`s for the root with the external resolver (i.e., `_` is not an actual dependency to add, but it specifies multiple "inner" dependencies that should be added).